modules/building-operator-with-helm-using-osdk.adoc

```diff
@@ -0,0 +1,352 @@
+// Module included in the following assemblies:
+//
+// * operators/osdk-helm.adoc
+
+[id='building-operator-with-helm-using-osdk_{context}']
+= Building an Operator with Helm charts using the Operator SDK
+
+This procedure walks through an example of building a simple Nginx Operator
+powered by a Helm chart using tools and libraries provided by the Operator SDK.
+
+[TIP]
+====
+It is best practice to build a new Operator for each chart. This can allow for
+more native-behaving Kubernetes APIs (e.g., `oc get Nginx`) and flexibility if
+you ever want to write a fully-fledged Operator in Go, migrating away from a
+Helm-based Operator.
+====
+
+.Prerequisites
+
+- Operator SDK CLI installed on the development workstation
+- Operator Lifecycle Manager (OLM) installed on a Kubernetes-based cluster (v1.8
+or above to support the `apps/v1beta2` API group), for example {product-title}
+4.0
+- Access to the cluster using an account with `cluster-admin` permissions
+- link:https://kubernetes.io/docs/tasks/tools/install-kubectl/[`kubectl`] v1.11.0+
```

Comment:
Based on [this comment](https://github.com/operator-framework/operator-sdk/pull/787#issuecomment-444638740) and the bug it references, I think the minimum Kubernetes cluster version and kubectl version is 1.11.3.